src/Mono.Android/Android.App/Activity.cs

```diff
@@ -22,6 +28,29 @@ public void RunOnUiThread (Action action)
 		{
 			RunOnUiThread (new Java.Lang.Thread.RunnableImplementor (action));
 		}
+
+		protected T SetContentView <T> () where T: LayoutBindingBase, new()
```

Comment:
If we're using `Activator.CreateInstance()` with multiple constructor arguments, why the `new()` constraint, which requires a default constructor?